### PR TITLE
[@types/jexcel] add freezeColumns 

### PR DESCRIPTION
**Pull request by LHTrungSkySP**
set freezeColumns

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

**Review by MysteryBlokHed — COMMENTED** [review]
If you update the tests ([`jexcel-tests.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jexcel/jexcel-tests.ts)), then this can be merged without needing a DefintelyTyped maintainer's review.

**Review by sheetalkamat — APPROVED** [review]

**Comment by MysteryBlokHed** [comment]
> If you update the tests ([`jexcel-tests.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jexcel/jexcel-tests.ts)), then this can be merged without needing a DefintelyTyped maintainer's review.

Not needed anymore, but I can re-approve if you decide to do this

**Pull request closed without merging**
